azure_billing/scrape.py

```diff
@@ -74,7 +74,7 @@ def data(enrollment, token, month=None):
 
     df = convert_json_df(raw)
 
-    c = Counter('azure_costs', "Costs billed to Azure Enterprise Agreement {}".format(enrollment))
+    c = Counter(metric_name, "Costs billed to Azure Enterprise Agreement {}".format(enrollment))
```

Comment:
You suggest to add the enrollment as label? Haven't thought about that yet. I guess it is rather rare to have several enterprise agreements per company. As naturally one should be enough.
One use case however is having an EA for billing in € and one for billing in $. 
The naming best practices would rather suggest to put that in the metric name, e.g. `azure_costs_eur` and `azure_costs_usd`. The mapping however cannot be made automatically. My idea was that this can be encoded in the configuration.